ansible/files/configuration/configuration.json

```diff
@@ -20,9 +20,11 @@
     "Properties": {
       "spark.local.dir": "/mnt,/mnt1",
       "spark.driver.maxResultSize": "4g",
+      "spark.driver.memory": "4g",
```

Comment:
Since we already set `"maximizeResourceAllocation": "true"` this value should be dynamically adjusted based on the instance type being used [1]. What is it being set to today on c3.4xlarge?

[1] http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-configure.html#d0e20480